pennylane/ops/op_math/condition.py

```diff
@@ -616,7 +615,7 @@ def wrapper(*args, **kwargs):
                     QueuingManager.remove(op)
 
             # 1. Apply true_fn conditionally
-            qscript = make_qscript(true_fn)(*args, **kwargs)
+            qscript = qml.tape.make_qscript(true_fn)(*args, **kwargs)

```

Comment:
And even more circular import issues. `qml.tape` should depend on `qml.ops`, not the other way around.